tools/py/mux-positions.py

```diff
@@ -53,8 +94,16 @@ def main(fns_in, fn_out, begin, end, stride):
 
         try:
             # Get the frames from all trajectories...
-            for trj in trjs_in:
+            for trj in trjs_in:                
                 frame = trj.next()
+
+                if wrap:
+                    frame = wrap_positions(frame)
+                if unwrap:
+                    frame = unwrap_positions(frame,frame_last)
+
+
+                frame_last = frame
```

Comment:
There can be multiple trajectories, so you should store a frame_last for each trajectory. 
Something like holding a list sized as trjs_in, and traversing in together with trjs_in.

Reply:
done.